src/server/game/Entities/Player/Player.cpp

```diff
@@ -9780,9 +9780,49 @@ void Player::ApplySpellMod(uint32 spellId, SpellModOp op, T& basevalue, Spell* s
         if (mod->type == SPELLMOD_FLAT)
         {
             // xinef: do not allow to consume more than one 100% crit increasing spell
-            if (mod->op == SPELLMOD_CRITICAL_CHANCE && totalflat >= 100)
+            if (mod->op == SPELLMOD_CRITICAL_CHANCE)
             {
-                return;
+                // Checks if the Juggernaut and Recklessness buffs are active
+                bool hasJuggernaut = HasAura(65156); // Buff Juggernaut
+                bool hasRecklessness = HasAura(1719); // Buff Recklessness
+
+                // Lista de IDs para Mortal Strike e Slam (Incluindo múltiplos ranks)
+                uint32 mortalStrikeRanks[] = { 12294, 21551, 21552, 21553, 25248, 30330, 47485, 47486 };  // Mortal Strike Rank IDs
+                uint32 slamRanks[] = { 1464, 8820, 11604, 11605, 25241, 25242, 47474, 47475 }; // Slam Rank IDs
```

Comment:
1. we use english
2. Just no. Do not ever hardcode raw values
3. Try to fix the issue in a spellscript

Reply:
Sorry. Ok